src/components/aws/map/MapComponent.js

```diff
@@ -41,7 +41,7 @@ const generateTooltip = (region, data) => {
 
   const costValue = document.createElement("div");
   costValue.classList.add("tooltip-cost-value");
-  costValue.innerHTML = parseFloat(data.total.toFixed(2)).toLocaleString();
+  costValue.innerHTML = (data.total < 0.01 && data.total > 0 ? "<0.01" : parseFloat(data.total.toFixed(2)).toLocaleString());
```

Comment:
Why not using the formatPrice formatters you modified above here ?

Reply:
Because `formatPrice` returns a React Element and not a string as required for `innerHTML`. I also tried to use `appendChild` but since it isn't a regular Node, it doesn't work.

Reply:
Ok, make sense. Thanks